frontend-react/src/hooks/UseOktaMemberships.ts

```diff
@@ -96,10 +103,15 @@ export const makeMembershipMapFromToken = (
 };
 
 const defaultState: MembershipState = {
+    // note that active will be set to {} rather than undefined in most real world cases on initialization
```

Comment:
I went back and forth on this a bit but where I ended up was:

* we're storing and reading a json string from session storage for the organization override, so returning a non-json value there would be breaking the expectation that we're getting JSON
* if we do return a falsey value from `getOrganizationOverride` we would need to throw some logic into `calculateMembershipsWithOverride` to account for that possibility, which complicates the function a bit
* there's no inherent value in having an undefined value for `active` at this point since we are, in all the cases in the code I've found, referring to a property of `active`, and thus sort of expecting that it's an object already

Definitely willing to put in that slightly more complex logic to end up with undefined if you think it's worth it.

Reply:
I guess it's a philosophical question of whether this backs us into difficulties with React/TS. Falsey values within rigidly defined interfaces are a big help in TypeScript React when data is constantly updating, and this seems like it'd create an _extra_ layer of complexity.

Reply:
Yeah, I think that's a fair point. I will refactor this to set `active` as undefined rather than `{}` if there is no override or active membership in state.